docs/moderne-cli/README.md

```diff
@@ -43,6 +43,10 @@ Boot 2 to Spring Boot 3.
    </TabItem>
    </Tabs>
 
+  :::tip
+  On MacOS you can also use homebrew: `brew install moderneinc/moderne/mod` and skip step 4.
+  :::
+
```

Comment:
Thanks for adding this! Can we move this into the Mac tab above? Then you can also drop the "On MacOS " prefix.